### PR TITLE
DOC: Fix git commands for creating your own library

### PR DESCRIPTION
**Pull request by bryandeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,13 +45,13 @@ tests under `template/tests` which can be run using `nosetests`.
...
 ```shell
-$ git clone https://github.com/vighneshbirodkar/project-template.git
+$ git clone https://github.com/scikit-learn-contrib/project-template.git
 ```
 You should rename the `project-template` folder to the name of your project.
 To host the project on Github, visit https://github.com/new and create a new
 repository. To upload your project on Github execute
 ```shell
-$ git remote add origin https://github.com/username/project-name.git
+$ git remote set-url origin https://github.com/username/project-name.git
```
Comment (vighneshbirodkar) [inline README.md:54]: @bryandeng Why do you suggest this change ?

**Comment by vighneshbirodkar** [comment]
@bryandeng Thank you for your contribution.

File: README.md (hunk 1 above)
Comment (bryandeng) [inline README.md:54]: `origin` points to https://github.com/scikit-learn-contrib/project-template.git after cloning.

So you'll get a fatal error when trying to do `git remote add origin`:

> fatal: remote origin already exists.

**Comment by vighneshbirodkar** [comment]
@bryandeng Can you rebase this so we can merge ? The tests pass on the latest code

**Comment by bryandeng** [comment]
@vighneshbirodkar OK. Done.

**Comment by vighneshbirodkar** [comment]
Thanks @bryandeng

**Pull request merged**